src/contractors.jl

```diff
@@ -2,125 +2,99 @@ export Contractor
 export Bisection, Newton, Krawczyk
 
 """
-    𝒩(f, f′, X, α)
-
-Single-variable Newton operator.
-
-The symbol for the operator is accessed with `\\scrN<tab>`.
-"""
-function 𝒩(f, f′, X::Interval{T}, α) where {T}
-    m = Interval(mid(X, α))
-
-    return m - (f(m) / f′(X))
-end
-
-"""
-    𝒩(f, jacobian, X, α)
+    Contractor{F}
 
-Multi-variable Newton operator.
+Abstract type for contractors.
 """
-function 𝒩(f::Function, jacobian::Function, X::IntervalBox, α)  # multidimensional Newton operator
-    m = Interval.(mid(X, α))
-    J = jacobian(X)
-    y = gauss_elimination_interval(J, f(m)) # J \ f(m)
-    return IntervalBox(m .- y)
-end
+abstract type Contractor{F} end
```

Comment:
how would you feel about renaming this to `AbstractContractor`?

Btw, while you are at it, we could close #128 and not export this?

Reply:
I'm fine with both.